src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -64,11 +94,12 @@ static std::vector<ngraph::Shape> get_numpy_broadcast_shape(ngraph::Shape left_s
 /// \param[in]  output_shape  The output shape.
 /// \param[in]  source_shape  The source shape from which we want to broadcast input node.
 ///
-/// \return     The boroadcasted Node.
+/// \return     The broadcasted Node.
 ///
-static std::shared_ptr<ngraph::Node> broadcast(const std::shared_ptr<ngraph::Node>& node,
-                                               const ngraph::Shape& output_shape,
-                                               const ngraph::Shape& source_shape)
+static std::shared_ptr<ngraph::Node>
+    broadcast_node_numpy_style(const std::shared_ptr<ngraph::Node>& node,
+                               const ngraph::Shape& output_shape,
+                               const ngraph::Shape& source_shape)
```

Comment:
Please add a note and a check that `source_shape` has same rank as `output_shape`.